src/git_changelog/build.py

```diff
@@ -137,6 +136,21 @@ def is_minor(self) -> bool:
         """
         return bool(self.tag.split(".", 2)[2])
 
+    def add_commit(self, commit: Commit) -> None:
+        """Adds the given commit and assigns it a section based on the chosen commit convention.
+
+        Arguments:
```

Comment:
Nitpick: I prefer "Parameters" over "Arguments". Not very important.

```suggestion
        Parameters:
```

Reply:
Nevermind, will change all occurrences in another PR.